_defaults.yml

```diff
@@ -2,21 +2,12 @@
 # role values. This is useful for "downstream"
 # packaging and opinionated releases.
 #
-# If in doubt, leave the file empty.
-#
-# Example:
-#
-# validator_namespace: myproject
-# validator_image: registry.example.com/mine/kubevirt-template-validator
-# validator_tag: v0.6.1
-# templates_version: v0.6.2
+# this file should contain only the tags (~= versions) of the containers to install.
```

Comment:
What do would you say to this?

```suggestion
# This file can be left empty to use the latest and greatest versions of all containers.
# Or it can contain the tags (~= versions) of the containers to install if there is a known combination you want.
# In case you really know what you are doing it is also possible to override the container names here.
```